src/components/Graphs/LineGraph.js

```diff
@@ -1,5 +1,5 @@
 import React, { PureComponent } from 'react';
-import { LineGraphStyle } from './LineGraphStyle';
+// import { LineGraphStyle } from './LineGraphStyle';
```

Comment:
General question: not 100% sure but I think we are not using the LineGraphStyle. If so shouldn't we just remove it?

Reply:
Yes, I will remove it!